### ISSUE TITLE
Let Mario use doors like in SM64

### ISSUE THREAD

**Issue by crzylemon** [Retro64Mod/Retro64Mod.github.io#198]
For example:

![image](https://github.com/Retro64Mod/Retro64Mod.github.io/assets/115381147/ec7dda19-dbd5-4511-ba29-e9be00340ecc)

**Comment by Mariosonicman91** [comment]
Is... this mod still being updated?

**Comment by Danmccool** [comment]
Retro 64 Is Getting A 1.21 Port Sometime

**Comment by Mariosonicman91** [comment]
YES! I really want to use Axiom with this mod and I aint paying for a commercial license just to play with a backported version.

**Comment by Mariosonicman91** [comment]
Quick question, do you know if Actors are being implemented?
As in, Goombas, Coins, and ! Blocks?
